tests/ethercat/test_pdo.py

```diff
@@ -296,6 +296,7 @@ def test_start_stop_pdo(connect_to_all_slave):
             rpdo_map.add_registers(register)
         for item in rpdo_map.items:
             item.value = new_operation_mode[index]
+        servo.reset_pdo_mapping()
```

Comment:
I think reset should de done in the Servo PDO constructor or similar. To ensure the inital state.

Reply:
In addition, you can add a test or expand this test to check that you can stop PDOs and start it again with the same configuration if you don't reset the mapping.